src/components/StatsGathering/ForceBrowserShareAssets.tsx

```diff
@@ -2,12 +2,14 @@
 /* eslint-disable jsx-a11y/alt-text */
 interface Props {
   visibleForces: string
+  userId: number
 }
 
-const ForceBrowserShareAssets = ({ visibleForces }: Props) => {
+const ForceBrowserShareAssets = ({ visibleForces, userId }: Props) => {
   const forces = visibleForces.split(",")
+  const id = userId.toString()
```

Comment:
I don't think you need to explicitly convert this; a) I think it's probably already a string and b) using it in a template string will automatically convert it